Source/Core/Core/PatchEngine.h

```diff
@@ -4,11 +4,15 @@
 
 #pragma once
 
+#include <map>
+#include <memory>
 #include <string>
 #include <vector>
 
 #include "Common/CommonTypes.h"
 
+#include "DiscIO/Volume.h"
```

Comment:
This can be a forward declaration (assuming the include would otherwise be required, despite migration of the patching code to the volume handling code)